### ISSUE TITLE
use light mode G not work at all

### ISSUE THREAD

**Issue by HUSTLX** [znxlwm/UGATIT-pytorch#28]
just like this, which D has a very small loss
![image](https://user-images.githubusercontent.com/12711390/63330818-0ede2c80-c367-11e9-943a-1bf358e90ccc.png)

**Comment by Frizy-up** [comment]
may need to train more steps

**Comment by HUSTLX** [comment]
> may need to train more steps

already, it dose not work

**Comment by saturosfz** [comment]
I also met such case

**Comment by cfanyyx** [comment]
+1, I already trained more than 100000 steps but still got blank outputs.

**Comment by xiaosean** [comment]
+1, Using light mode gets similar results.

**Comment by cfanyyx** [comment]
It seems the input channel after adaptive_avg_pooling in light mode can't be set too small.
`if self.light:
            FC = [nn.Linear(ngf * mult, ngf * mult, bias=False),
                  nn.ReLU(True),
                  nn.Linear(ngf * mult, ngf * mult, bias=False),
                  nn.ReLU(True)]`
if I change the first **ngf * mult** to a larger one. Then the output img is normal.

**Comment by jayzhan211** [comment]
I try with this command
`python main.py --dataset selfie2anime --light True --resume True --save_freq 10000 --benchmark_flag True`
and the following is my result at 100,000 epoch.  btw I continue the training start from 10000 (it is just the save network testing before).
![](https://i.imgur.com/JoD8zxh.png)
![](https://i.imgur.com/VZ0fmcf.png)

B2A is terrible...
